Resources/Textures/ADT/Structures/coffeemachine.rsi/meta.json

```
@@ -0,0 +1,46 @@
+{
+  "version": 1,
```

Версия номер 222222222
```suggestion
  "version": 2,
```